pkgs/games/osu-lazer/default.nix

```diff
@@ -16,13 +16,13 @@ let
 
 in stdenv.mkDerivation rec {
   pname = "osu-lazer";
-  version = "2021.703.0";
+  version = "2021.707.0-lazer";
```

Comment:
```suggestion
  version = "2021.707.0";
```

It tagged both `2021.707.0` and `2021.707.0-lazer` to the same commit. I'm not sure why there are two but we should not include this suffix in `version`.

Reply:
Yeah, I was confused by that too. Since they're same commit, I think we should do that.